src/main/java/com/jspiders/shoppingcart/dao/AdminDao.java

```diff
@@ -0,0 +1,19 @@
+package com.jspiders.shoppingcart.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.jspiders.shoppingcart.dto.Admin;
+import com.jspiders.shoppingcart.dto.Customer;
+import com.jspiders.shoppingcart.dto.Merchant;
+
+public interface AdminDao {
+
+	public Admin saveAdmin(Admin admin);
```

Comment:
we can avoid writing public right?
and merchants i have created in merchant DAO.
i think here only admin save and validate will be enough

Reply:
ok

Reply:
wt abt customer

Reply:
Customer also done with the name findCustomerById and fetchAllCustomers in customer.